test/suites/common/xcm/test-mock-dmp-transact.ts

```diff
@@ -0,0 +1,120 @@
+import { beforeAll, describeSuite, expect, customDevRpcRequest } from "@moonwall/cli";
+import { alith } from "@moonwall/util";
+
+import { ApiPromise } from "@polkadot/api";
+import {
+  XcmFragment,
+  injectDmpMessageAndSeal,
+  descendParentOriginFromAddress32,
+  descendParentOriginForAddress20
+} from "../../../util/xcm.ts";
+import { generateKeyringPair } from "@moonwall/util";
+import { expectOk } from "../../../util/expect.ts";
+import { Keyring } from "@polkadot/api";
+import { BN } from "@polkadot/util";
+
+describeSuite({
+  id: "D1",
+  title: "Mock XCM - Succeeds using sovereign accounts",
+  foundationMethods: "dev",
+  testCases: ({ context, it, log }) => {
+    let polkadotJs: ApiPromise;
+    let transferredBalance;
+    let sendingAddress;
+    let descendAddress;
+    let alice;
+    let chain;
+
+    beforeAll(async function () {
+        polkadotJs = context.polkadotJs();
+        chain = polkadotJs.consts.system.version.specName.toString();
+        alice = chain == 'frontier-template' ? alith : (new Keyring({ type: 'sr25519' }).addFromUri('//Alice', { name: 'Alice default' }));
+        let descendFunction = chain == 'frontier-template' ? descendParentOriginForAddress20 : descendParentOriginFromAddress32;
+
+        const { originAddress, descendOriginAddress } = descendFunction(context);
+
+        sendingAddress = originAddress;
+        descendAddress = descendOriginAddress;
+
+        transferredBalance = 10_000_000_000_000n;
+
+        const txSigned = polkadotJs.tx.balances.transfer(descendOriginAddress, transferredBalance);
+
+        await expectOk(
```

Comment:
`expectOk` is no needed here, if you want you can use the `{ allowFailures: false }` option inside `createBlock()` function instead.